packages/studio-web/src/app/demo/demo.component.html

```diff
@@ -12,7 +12,7 @@
         </h2>
       </div>
       <div class="col-3 download__buttons">
-        <mat-form-field appearance="fill">
+        <mat-form-field class="d-none d-lg-block" appearance="fill">
```

Comment:
That's a good idea, it makes no sense to allow all the output formats on a mobile device.

Question of alignment: I would like the download icon to be at the far right when the download types are not there. I tried to edit `demo.component.sass` to do it, but I failed to move that darn thing. Not critical, but it would be nice to have the download__buttons div float to the right if you can figure out how to do it right.